src/core/services/layout/layout.scss

```diff
@@ -512,6 +514,7 @@
 @media (min-width: $layout-breakpoint-lg) {
   @include layouts_for_breakpoint(gt-lg);
   @include layouts_for_breakpoint(xl);
+  @include layouts_for_breakpoint(print);
```

Comment:
@ThomasBurleson I forgot to remove that sorry.

I was having problems because of the ie hack using `screen\0` [here](https://github.com/angular/material/pull/6784/files#diff-afb8b370e6204859a801fc83d3b0f2f4R145). It basically returned from that block when it got to the screen part for the `print` styles. I didn't know this so I tried to get print styles working for something but I must of forgotten about it.

I removed it in a commit below.